scipy/stats/tests/test_multivariate.py

```diff
@@ -810,6 +1038,67 @@ def test_cdf_signs(self):
         cdf = multivariate_normal.cdf(b, mean, cov, lower_limit=a)
         assert_allclose(cdf, cdf[0]*expected_signs)
 
+    @pytest.mark.slow
+    def test_cdf_vs_cubature(self):
+        ndim = 3
+        rng = np.random.default_rng(123)
+        a = rng.uniform(size=(ndim, ndim))
+        cov = a.T @ a
+        dist = multivariate_normal(np.zeros(ndim), cov=cov)
```

Comment:
Could we do a nonzero mean in this test, at least?

Reply:
Done.